analyzer/src/funTest/assets/projects/external/multi-kotlin-project-expected-output-cli.yml

```diff
@@ -15,95 +15,222 @@ project:
     revision: "<REPLACE_REVISION>"
     path: "analyzer/src/funTest/assets/projects/external/multi-kotlin-project/cli"
   homepage_url: ""
-  dependency_graph:
-    packages:
-    - "Gradle:org.gradle.kotlin.dsl.samples.multiproject:core:1.0"
-    - "Maven:org.jetbrains.kotlin:kotlin-stdlib:1.3.11"
-    - "Maven:org.jetbrains.kotlin:kotlin-stdlib-common:1.3.11"
-    - "Maven:org.jetbrains:annotations:13.0"
-    - "Maven:org.jetbrains.kotlin:kotlin-compiler-embeddable:1.3.11"
-    - "Maven:org.jetbrains.kotlin:kotlin-script-runtime:1.3.11"
-    - "Maven:org.jetbrains.kotlin:kotlin-reflect:1.3.11"
-    - "Maven:org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.3.11"
-    scope_roots:
-    - dependencies:
-      - pkg: 1
+  scopes:
```

Comment:
Hint:

> When support for the DependencyGraph format was added to Gradle in
c8d69ca, the expectations of tests
have been adapted to the new format. This is going to be problematic

The formatting looks odd here as GitHub shortens commit SHA1 to 7 chars. You can align the plain text and rendered view by also only using 7 chars in the original commit message.